script.js

```diff
@@ -1,129 +1,159 @@
-    $commentContainerFormControls.show();
+  if (commentContainerTextarea && commentContainerTextarea.value !== '') {
+    commentContainerFormControls.style.display = 'block';
   }
 
   // Expand Request comment form when Add to conversation is clicked
-  var $showRequestCommentContainerTrigger = $(".request-container .comment-container .comment-show-container"),
-    $requestCommentFields = $(".request-container .comment-container .comment-fields"),
-    $requestCommentSubmit = $(".request-container .comment-container .request-submit-comment");
-
-  $showRequestCommentContainerTrigger.on("click", function() {
-    $showRequestCommentContainerTrigger.hide();
-    $requestCommentFields.show();
-    $requestCommentSubmit.show();
-    $commentContainerTextarea.focus();
+  var showRequestCommentContainerTrigger = document.querySelector('.request-container .comment-container .comment-show-container'),
+    requestCommentFields = document.querySelectorAll('.request-container .comment-container .comment-fields'),
+    requestCommentSubmit = document.querySelectorAll('.request-container .comment-container .request-submit-comment');
+
+  showRequestCommentContainerTrigger && showRequestCommentContainerTrigger.addEventListener('click', function() {
+    showRequestCommentContainerTrigger.style.display = 'none';
+    requestCommentFields.forEach(e => e.style.display = 'block');
+    requestCommentSubmit.forEach(e => e.style.display = 'block');
+    commentContainerTextarea && commentContainerTextarea.dispatchEvent(new Event('focus'));
   });
 
   // Mark as solved button
-  var $requestMarkAsSolvedButton = $(".request-container .mark-as-solved:not([data-disabled])"),
-    $requestMarkAsSolvedCheckbox = $(".request-container .comment-container input[type=checkbox]"),
-    $requestCommentSubmitButton = $(".request-container .comment-container input[type=submit]");
-
-  $requestMarkAsSolvedButton.on("click", function () {
-    $requestMarkAsSolvedCheckbox.attr("checked", true);
-    $requestCommentSubmitButton.prop("disabled", true);
-    $(this).attr("data-disabled", true).closest("form").submit();
+  var requestMarkAsSolvedButton = document.querySelector('.request-container .mark-as-solved:not([data-disabled])'),
+    requestMarkAsSolvedCheckbox = document.querySelector('.request-container .comment-container input[type=checkbox]'),
+    requestCommentSubmitButton = document.querySelector('.request-container .comment-container input[type=submit]');
+
+  requestMarkAsSolvedButton && requestMarkAsSolvedButton.addEventListener('click', function () {
+    requestMarkAsSolvedCheckbox.setAttribute('checked', true);
+    requestCommentSubmitButton.disabled = true;
+    this.setAttribute('data-disabled', true);
+    // Element.closest is supported in modern browsers and has a polyfill available for still in use outdated versions
+    // See https://caniuse.com/#feat=element-closest
+    // TODO: Should polyfill be installed? 
```

Comment:
I don't think we need this comment here.
Instead, above the `closest` function declaration we could simply put a `Element.closest not yet supported in IE11`.